src/VisualStudio/Xaml/Impl/project.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "dependencies": {
+    "NuGet.VisualStudio": {
```

Comment:
Actually, make that:
    "Microsoft.VisualStudio.Editor": "14.3.25407",
    "Microsoft.VisualStudio.Shell.15.0-pre": "15.0.25511.2-pre"